enterprise/admin/utils.py

```diff
@@ -28,8 +38,19 @@ class ValidationMessages(object):
     BOTH_FIELDS_SPECIFIED = _("Either \"Email or Username\" or \"CSV bulk upload\" must be specified, but both were.")
     BULK_LINK_FAILED = _("Bulk operation failed - no users were linked. Please correct the errors listed below.")
     COURSE_MODE_INVALID_FOR_COURSE = _("Enrollment mode {course_mode} not available for course {course_id}.")
-    COURSE_WITHOUT_COURSE_MODE = _("Please select a course enrollment mode for the given course.")
+    COURSE_WITHOUT_COURSE_MODE = _("Please select a course enrollment mode for the given course/program.")
     INVALID_COURSE_ID = _("Could not retrieve details for the course ID {course_id}. Please specify a valid ID.")
+    INVALID_PROGRAM_ID = _(
```

Comment:
I go back and forth on whether it's better to alphabetize or group by concept -- we need to do it one way or the other, however, and alphabetical is the current convention, so maybe we should stick with that.